addOns/commonlib/src/main/javahelp/help/contents/alerttags.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>
+        Alert Tags
+    </title>
+</head>
+<body>
+<h1>Alert Tags</h1>
+
+The Common Library add-on provides Alert Tags for use by any or all scan rules.
+
+<p>
+Of note the following tags/groups of tags are included:
+<ul>
+    <li>Custom Payloads &rarr; A tag which indicates the scan rules which support <a href="https://www.zaproxy.org/docs/desktop/addons/custom-payloads/">Custom Payloads functionality</a>.</li>
+    <li>HIPAA (Health Insurance Portability and Accountability Ac) &rarr; Tags representing alerts/rules which we've mapped to the HIPAA standard.</li>
+    <li>OWASP Top 10 - 2017 &rarr; Tags representing the risks/vulnerabilities from the 2017 OWASP Top 10 list.</li>
+    <li>OWASP Top 10 - 2021 &rarr; Tags representing the risks/vulnerabilities from the 2021 OWASP Top 10 list.</li>
+    <li>PCI DSS (Payment Card Industry Data Security Standard) &rarr; Tags representing alerts/rules which we've mapped to the PCI DSS standard.</li>
```

Comment:
There's only one tag, is this plural on purpose? I see you added to all of them regardless of how many.

Reply:
No it wasn't on purpose, I guess I was just thinking that none of the tags apply to a single rule. But you're right it does make more sense if a bunch of them are singular.